bftengine/src/bftengine/ViewsManager.cpp

```diff
@@ -736,7 +753,10 @@ bool ViewsManager::tryMoveToPendingViewAsPrimary(ViewNum v) {
     if (relatedVCMsgs.size() == SMAJOR) break;
   }
 
-  if (relatedVCMsgs.size() < SMAJOR) return false;
+  if (relatedVCMsgs.size() < SMAJOR) {
+    LOG_INFO(GL, "Waiting for sufficient ViewChange messages to entering view as primary ...");
```

Comment:
Maybe for some of those log messages we can continue the tradition started by Guy - adding "*******" to highlight them in the logs? Probably not for all of the logs you've added - but at least the most important ones.

Reply:
I think it is good to have consistency in the logs for View change, will add the prefix. Thanks for the remark.